android/src/main/kotlin/com/example/user_location_plugin/UserLocationPlugin.kt

```diff
@@ -1,35 +1,143 @@
 package com.example.user_location_plugin
 
+import android.Manifest
```

Comment:
Try to do Ctrl + Alt + L on this file, some things on the file looked not indented

Reply:
done